File: src/main/java/project/rpg/player/status/thing/Handicraft.java
package project.rpg.player.status.thing;

import org.bukkit.attribute.Attribute;
import org.bukkit.entity.Player;
import project.rpg.player.status.base.StatusBase;
import project.rpg.player.status.base.StatusName;

public class Handicraft extends StatusBase{   //손재주

    public Handicraft() {
        super(StatusName.HANDICRAFT.getName());
    }

    @Override
    public void effect(Player player) {
    }

}

File: src/main/java/project/rpg/player/status/objects/Handicraft.kt
package project.rpg.player.status.objects

import kotlinx.serialization.Serializable
import org.bukkit.entity.Player
import project.rpg.player.status.base.StatusBase
import project.rpg.player.status.base.StatusName

@Serializable
class Handicraft: StatusBase(StatusName.HANDICRAFT) {
    override fun effect(player: Player?) {
        TODO("구현 예정")
    }
}